src/twisted/internet/defer.py

```diff
@@ -160,6 +160,8 @@ def maybeDeferred(f, *args, **kw):
 
 
 
+@deprecated(Version('Twisted', 16, 8, 0),
```

Comment:
Presumably this will be a 17.x.y now but I'm not sure what the current system for picking such numbers is, nor how one predicts this in advance. I heard some folks talking about `Version("Twisted", "NEXT", 0, 0)`?  Maybe that was a joke of some sort though.

Probably at minimum bump this to some 17.x.y guess.